### ISSUE TITLE
More explicit error after attempting to scale constant in nnetar function

### ISSUE THREAD

**Issue by raphifix** [robjhyndman/forecast#681]
I accidentally included a matrix of external regressors that contained a column of zeros and was greeted with the following error:
```
Error in rowMeans(sapply(fit, predict)) : 
  'x' must be an array of at least two dimensions
```

It took me longer than I'd like to admit to determine that `scale.inputs` was left in its default `TRUE` and as such, was turning the column into `NaNs`.

Would it be possible to do a check for constants prior to scaling and give a more explicit error message to the user?

**Comment by raphifix** [comment]
Totally forgot to give an example. Thank you!

**Comment by mitchelloharawild** [comment]
Resolved in #667 
A warning is now provided and `scale.inputs` is set to `FALSE`
